docs/backend/guides/how-to/upgrade-go-version.md

```diff
@@ -58,20 +58,20 @@ go version go1.16.4 darwin/amd64
     - `scripts/gen-assets`
     - `scripts/gen-server`
     - `scripts/pre-commit-swagger-validate`
-    - `GOVERSION` in `scripts/prereqs`
+    - `scripts/run-e2e-test-docker`
     - `Makefile` the `docker_circleci` task
     - `nix/default.nix` (you will also need to [update the hash](https://ahobson.github.io/nix-package-search))
+    - .go-version
```

Comment:
I also wonder if the `nix/default.nix` bit should be moved to be in the previous section where the `.tool-versions` is since a `nix` user would do that instead to update their local `go` version.